NitroxServer/Serialization/ServerConfig.cs

```diff
@@ -30,6 +30,9 @@ public class ServerConfig : NitroxConfig<ServerConfig>
         [PropertyDescription("Prevents players from losing items on death")]
         public bool KeepInventoryOnDeath { get; set; } = false;
 
+        [PropertyDescription("Places a beacon where players die")]
+        public bool MarkDeathPointsWithBeacon { get; set; } = false;
```

Comment:
Shouldn't the default value for this be set to true? I feel like it'd be more useful that way than forcing people to turn the setting on.

Reply:
I agree it should be true by default